spec/features/photos_feature_spec.rb

```diff
@@ -0,0 +1,136 @@
+require 'rails_helper'
+
+feature 'photos' do
+
+  context 'no photo have been added' do
+    scenario 'should display a prompt to add a photo' do
+      visit '/photos'
+      expect(page).to have_link 'Add a photo'
+    end
+  end
+
+  context 'adding photo' do
+
+    scenario 'should display a photo and his title' do
+      visit '/photos'
+      sign_up
+      click_link 'Add a photo'
+      attach_file 'Image', 'spec/features/sea.jpg'
+      fill_in 'Title', with: 'Awesome sea'
+      click_button 'Add photo'
+      expect(page).to have_content 'Awesome sea'
+      expect(page).to have_css('img[src*="sea.jpg"]')
+    end
+
+    scenario 'does not let you submit a photo without a title' do
+      visit '/photos'
+      sign_up
+      click_link 'Add a photo'
+      attach_file 'Image', 'spec/features/sea.jpg'
+      fill_in 'Title', with: ''
+      click_button 'Add photo'
+      expect(page).to have_content 'error'
+    end
+
+    scenario 'does not let you submit a photo without upload a image' do
+      visit '/photos'
+      sign_up
+      click_link 'Add a photo'
+      fill_in 'Title', with: 'hello world'
+      click_button 'Add photo'
+      expect(page).to have_content 'error'
+    end
+
+
+  end
+
+  context 'viewing photos' do
+
+    let!(:sea) {Photo.create(title:'Awesome sea',
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.